src/components/page-elements/info.vue

```diff
@@ -1,13 +1,7 @@
 <template>
-  <div :class="['info', `info--${type}`]" class="info__container m-3">
+  <div :class="['success', `success--${type}`]" class="info__container m-3">
```

Comment:
Dlaczego success? To jest komponent info.vue a nie success.vue. Success jest typem a nie komponentem. Proszę przywrócić klasę "info".